### ISSUE TITLE
[Enhancement]: Hide sub series in the general library view 

### ISSUE THREAD

**Issue by Hallo951** [advplyr/audiobookshelf#2092]
### Describe the feature/enhancement

I have an idea how to solve the problem with the subseries in a simple way.

Currently, I do it in such a way that I also enter the main series in the "Series" field for each sub-series. This has the effect that all sub-series are displayed within the main series when the "collapse series" option is activated.

The problem with this solution is that in this case the subseries are shown twice in the library view. Once within the main series and once as independent series in the general library view.

One solution to this problem would be to mark the sub-series. To do this, one would have to enter a kind of code word, e.g. "sub-series", in the series input field where one enters the series name(s) for an audio book. This firmly defined code word causes Abs not to create a new series, but to remove the series marked in this way from the general library view but not from the main series. The series marked in this way would then only be visible within the main series and would no longer appear twice in the database.

**Comment by hobesman** [comment]
Or you could have another toggle: "collapse series [  ]" and then if that's checked, "show sub series [  ]"

**Comment by KaiStarkk** [comment]
Along with #870, this enhancement would appear to require either a change to the data model, so that series can contain other series, or a workaround in the form of an additional property on a series marking it as a subseries.

The scanner could correspondingly mark this property positive, when the book is found in a subdirectory below the series level. (c.f. https://github.com/advplyr/audiobookshelf/issues/870#issuecomment-1834780827)

**Comment by kpalanick** [comment]
Is this something that is waiting of someone to work on? I could take this on, as I'd love this feature in audiobookshelf, and it doesn't seem to be urgent, so I can take my time developing it

**Comment by ElDubsNZ** [comment]
> Is this something that is waiting of someone to work on? I could take this on, as I'd love this feature in audiobookshelf, and it doesn't seem to be urgent, so I can take my time developing it

Yes please, plenty of us who listen to things like Big Finish would absolutely love this.

**Comment by epipenRX** [comment]
> ### Describe the feature/enhancement
> I have an idea how to solve the problem with the subseries in a simple way.
> 
> Currently, I do it in such a way that I also enter the main series in the "Series" field for each sub-series. This has the effect that all sub-series are displayed within the main series when the "collapse series" option is activated.
> 
> The problem with this solution is that in this case the subseries are shown twice in the library view. Once within the main series and once as independent series in the general library view.
> 
> One solution to this problem would be to mark the sub-series. To do this, one would have to enter a kind of code word, e.g. "sub-series", in the series input field where one enters the series name(s) for an audio book. This firmly defined code word causes Abs not to create a new series, but to remove the series marked in this way from the general library view but not from the main series. The series marked in this way would then only be visible within the main series and would no longer appear twice in the database.

i accidently opened a similar ticket. I searched for similar language but was unable to find it. i just want to post the notes i put together which are conceptually very similar to what you're describing. 
The main difference is i suggested:
- marking the MAIN-series rather than marking subsequent sub-series (mark one item instead of several) 
-  in my ticket i was specifically discussing the "series-tab view" where as you are discussing the "general-library-tab". 
 - i don't seem to see this problem in the current version on the general library tab (duplicate books due to belonging to multiple-series). I assume for this reason this was either a bug and was resolved, or you also meant to report the problem/enhancement as part of the "series-tab" too.

I hope these notes are useful


[Enhancement]: Option to View only Main-Series from Series-Tab (For series' that have multiple mini-series in the same primary-series-universe) #4448 
Type of Enhancement
Web Interface/Frontend

Describe the Feature/Enhancement
The ability to have only the "primary series" of a novella show up in the series tab.
For example:
Terry Pratchet or Dune books have an overarching Discworld or Dune Universe, but these books are made up of mini series that have their own series sequence.

It's great that ABS can allow you to assign multiple series sequences to books so in the case of discworld you can indicate the overarching order of the books, and subsequently also indicate the order of the mini-series that have their own sequence order.

PROBLEM: Because of this feature however, using discworld as the example, when you go to the series tab, you will see the master series "discworld" and also each subsequent mini-series as it's own "series" which saturates the "series" tab with the mini-series and adds a cluttered effect when viewing your library in this way.
Solution:

- it would be good if there were an (enable) option to indicate the master-series, and have an option to display only the master series from the "series" tab, or (disable) optionally display all series, including the master and mini-series displayed on the "series" tab. (enable/disable)
  - with only the master-series display-option selected: in the "series-tab", clicking on the master series could bring you to a page of mini-series' that exist within the master-series, with the master-series-sequence being the first selectable option, and subsequent mini-series appearing below

Why would this be helpful?
Removing repeated content will give the "series" tab a less-cluttered feel.

Because of the (very much appreciated) feature that allows for adding books to multiple-series-sequences, using discworld as the example, when you go to the series tab, you will see the master series "discworld" and also each subsequent mini-series as it's own "series" which saturates the "series" tab with the mini-series' and adds a cluttered effect when viewing your library in this way.

By adding a feature to optionally only show the "master-series" in the series tab (or be able to expand/collapse it somehow), it gives navigating by series a less-cluttered feel, and a more obvious visual navigation of the series collection as a whole.

Future Implementation (what would it look like)
Identify the master series in the book settings

- Add an option in the settings to display only the master-series from the "series-tab"

With the option enabled:

- at the "series tab" you see only the master-series (which has to be defined with some kind of check box when assigning a book to a series)
- A book can only be assigned 1 master-series,
  - once defined, the check box for "master-series" on other mini-series might be grayed out or invisible.
  - or, attempting to identify another series for the same book (that has a master-series defined as the master) would display an error indicating there can only be 1 master-series and tell the user what the name of the master-series is in the error message.
- Clicking on the master-series will bring you to a page of mini-series (from that master-series), with the first choice being the master-series, and the mini-series' shown after.
- Clicking on the master-series, or any mini-series, from here will bring you to the book-sequence from that mini-series (like normal)

With the option disabled:

- Audiobookshelf behavior at the Series-tab view behaves as it does in the current version 2.25.1

Audiobookshelf Server Version
2.25.1

Current Implementation (what page were you on when you thought of this)
"series" tab
- seeing a couple of series that are cluttering up the tab because they have several mini-series within the main-universe-series. Thought it would improve the view if people only saw the main-universe-series from that tab, but didn't want to remove information about the sub-series from the books.
